src/main/resources/administration.vm

```diff
@@ -403,32 +403,32 @@
                     <input class="text" type="text" id="scheduling-inactive-time" name="scheduling-inactive-time"
                            title="scheduling-inactive-time" placeholder="e.g.: [Inactive Time in days] ...">
                     <div class="description">
-                        After which time will a Timesheet be marked as Auto Inactive? (In Weeks)
+                        Number of weeks without a Timesheet entry after which a Timesheet will be marked as "Auto Inactive".
                     </div>
                 </div>
                 <div class="field-group">
                     <label for="scheduling-offline-time">Disabled Time</label>
                     <input class="text" type="text" id="scheduling-offline-time" name="scheduling-offline-time"
                            title="scheduling-offline-time" placeholder="e.g.: [Offline Time in days] ...">
                     <div class="description">
-                        After which time will a Timesheet be marked as Disabled? (In Weeks)
+                        Number of week without a Timesheet entry after which a Timesheet will be marked as "Disabled"
```

Comment:
Please change to "Number of weeks" (I have updated the ticket as well).

Reply:
Resolved in latest commit!